hera_mc/cm_handling.py

```diff
@@ -382,6 +392,34 @@ def show_parts(self, part_dossier):
             table_data.append(part_dossier[hpnr].table_entry_row(columns))
         print('\n' + tabulate(table_data, headers=headers, tablefmt='orgtbl') + '\n')
 
+    def get_physical_connections(self, at_date=None):
+        """
+        Finds and returns a list of "physical (@)" connections.
```

Comment:
I'm not sure what this line in the docstring means. Specifically what does the '(@)' mean? And maybe a little more clarity around what "physical" means (why is it in quotes?)

Reply:
I made the requested changes and threw in a few more tests within cm_handling